src/index.ts

```diff
@@ -820,21 +898,22 @@ export function HTPipe(): {};
 export function HTPipe<
   T extends OptionallyHasInitPreContext<any, any> &
     OptionallyHasSanitizeParams<any, any> &
+    OptionallyHasSanitizeQueryParams<any, any> &
     OptionallyHasSanitizeBody<any, any> &
     MightHavePreAuthorize<any, any> &
     OptionallyHasAttachData<any, any> &
     MightHaveFinalAuthorize<any, any> &
     OptionallyHasDoWork<any, any> &
     MightHaveRespond<any, any> &
     MightHaveSanitizeResponse<any, any>
```

Comment:
why remove the fixme??
- [ ] O.O

Reply:
because i think it's already done i previous PRs
AllStageKeys it's a union type of all lifecycle stages
and it looks like this:
export type AllAsyncStageKeys = 'attachData' | 'finalAuthorize' | 'doWork';
export type AllSyncStageKeys =
  | 'initPreContext'
  | 'sanitizeParams'
  | 'sanitizeQueryParams'
  | 'sanitizeBody'
  | 'preAuthorize'
  | 'respond'
  | 'sanitizeResponse';
export type AllStageKeys = AllAsyncStageKeys | AllSyncStageKeys;
so it was all this unions in Pick
but now it's just a type imported from types file

Reply:
yup you're right - i thought I'd written it for all the Optionally and Might things above ;)